scripts/build/termux_create_pacman_subpackages.sh

```diff
@@ -107,6 +107,10 @@ termux_create_pacman_subpackages() {
 			if [ -n "$TERMUX_SUBPKG_CONFFILES" ]; then
 				tr ',' '\n' <<< "$TERMUX_SUBPKG_CONFFILES" | awk '{ printf "backup = '"${TERMUX_PREFIX:1}"'/%s\n", $1 }'
 			fi
+
+			if [ -n "$TERMUX_PKG_GROUPS" ]; then
+				tr ',' '\n' <<< "${TERMUX_PKG_GROUPS/#, /}" | awk '{ printf "group = %s\n", $1 }'
```

Comment:
Why you are adding this only for subpackages?

Reply:
I only added subpackages so that the groups only have the basics of the packages.

Reply:
This means you will not be able to specify `automake`, `autoconf` and other packages as `base-devel` or `bash`, `coreutils` as `base` - just for example. Setting groups only for splits will make them barely useless.

Reply:
Ok